views/items/index.pug

```diff
@@ -25,14 +25,33 @@ block content
     .form-group
       label Select Parent
       select.form-control(data-style='btn', name='parent_id')
-        option(value='0') None
         each item in items
           option(value=`${item.id}`)= item.title
 
     button.btn.btn-primary(type='submit') Submit
 
+  ol.breadcrumb
+    li
+      a(href=`/items/${breadcrumbs[0].id}`)= breadcrumbs[0].title
+
+  each crumb in breadcrumbs.slice( 1, breadcrumbs.length - 1 )
+    li
+      a(href=`/items/${crumb.id}`)= crumb.title
+
   div
     h3 Your ToDo's:
 
-    .item-page
-      +display( tree, 0 )
+
```

Comment:
Remove extraneous whitespace - only one blank line is needed between logical blocks.